bot/cogs/image_dots_cog.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import urllib.request
+import random
+from PIL import Image
+from PIL import UnidentifiedImageError
+import math
+import requests
+
+import discord
+import discord.ext.commands as commands
+from bot.consts import Colors
+import bot.extensions as ext
+
+log = logging.getLogger(__name__)
+
+class DotCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.ASCIIYDOTS = 4
+        self.ASCIIXDOTS = 2
+        self.THRESHOLD = int(150)
```

Comment:
what does this int cast do here?

Reply:
It was something I had from the original code. Just lingering dead code statements. I will remove.